internal/config/mount_config.go

```diff
@@ -46,16 +46,16 @@ type MountConfig struct {
 // 3. compress: indicates whether the rotated log files should be compressed
 // using gzip. The default value is False.
 type LogRotateConfig struct {
-	MaxFileSizeMB int  `yaml:"max-file-size-mb"`
-	FileCount     int  `yaml:"file-count"`
-	Compress      bool `yaml:"compress"`
+	MaxFileSizeMB   int  `yaml:"max-file-size-mb"`
+	BackupFileCount int  `yaml:"backup-file-count"`
```

Comment:
nit: Please update the doc for backup-file-count. Also, mention the beahviour when backup-file-count = 0.

Reply:
Updated the documentation comments on top of this config. We are yet to publish public documentation for this. I will work with @cbonnie for the same.